### PR TITLE
Implement is_probable_cve query param

### PR DESCRIPTION
**Pull request by arajkumar**
Also moved string sanitiser as separate module.

### CODE REVIEW

File: src/sanitizer.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+"""Helper functions to sanitize/unsanitize str, List[str]"""
+
+# (fixme) remove once we move to gremlin driver
+
+from typing import Union, List
+from urllib.parse import quote, unquote
+
+def sanitize(string: Union[str, List[str]]) -> str:
```
Comment (rajusem) [inline src/sanitizer.py]: For my knowledge, if you don't do sanitize/unsanitize then is it failing while querying into graph db ? and to overcome we have aded this feature, is my understanding correct ?

Comment (arajkumar) [reply src/sanitizer.py]: It is not failing. Since we generate gremlin query string by appending data from user input, it is better to sanitize it to avoid query injection.

Comment (rajusem) [reply src/sanitizer.py]: got it, thanks

File: tests/src/sanitizer_test.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+"""To test sanitizer"""
+from src.sanitizer import sanitize, unsanitize
+
+# pylint: disable=missing-function-docstring
+def test_string_sanitization_with_empty():
```
Comment (rajusem) [inline tests/src/sanitizer_test.py]: Add test case for None/null condition
